github3/repos/release.py

```diff
@@ -227,7 +227,7 @@ def download(self, path=''):
             }
         resp = self._get(self._api, allow_redirects=False, stream=True,
                          headers=headers)
-        if resp.status_code == 302:
+        if resp and resp.status_code == 302:
```

Comment:
So `_get` and the other methods that proxy to requests always return a Response (unless requests itself raises an exception). We shouldn't need to check for None here since the only time we won't have a Response is in the exceptional case in which case we also won't be exercising this code. :)